.github/workflows/reuse.yaml

```diff
@@ -102,27 +102,32 @@ jobs:
         id: check-branch
         run: |
           if [[ ${{ steps.extract_branch.outputs.branch }} =~ ^release-[0-9]+\.[0-9]$ ]]; then
-              echo ::set-output name=match::true
-          fi
+                echo ::set-output name=match::true
+            fi
 
       - name: ko build dev image
         id: ko-publish-dev
         if: ${{inputs.tag == 'image' && steps.extract_branch.outputs.branch == 'main'}}
         run: |
+          set -e
           echo "::set-output name=digest::$(REGISTRY=ghcr.io REGISTRY_PASSWORD=${{secrets.registry_password}} make ${{inputs.publish_command}}-dev)"
 
       - name: ko build release image
         id: ko-publish
+        env:
+          COSIGN_REPOSITORY: "ghcr.io/${{ github.repository_owner }}/sbom"
         if: ${{inputs.tag == 'release' || (inputs.tag == 'image' && steps.check-branch.outputs.match == 'true')}}
         run: |
+          set -e
```

Comment:
These seem to be unnecessary?

Reply:
if we don't set this option, workflow looks like being succeed even if there is an error:

_https://github.com/developer-guy/kyverno/actions/runs/3101006912/jobs/5021906923#step:16:130_